ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-openshift-logging-6.0.yaml

```diff
@@ -0,0 +1,91 @@
+base_images:
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.17"
```

Comment:
```suggestion
  tests-private:
    name: tests-private
    namespace: ci
    tag: "4.16"
  tests-private-ui:
    name: tests-private-ui
    namespace: ci
    tag: "4.16"
```

Reply:
@anpingli FYI, you have to run `make update` in your local and commit the auto-generated changes to this PR.